docs/hello-signing.md

```diff
@@ -72,48 +72,54 @@ ACME Rockets will only deploy software that's been scanned and approved by the A
 - List the image, and any associated signatures
 
   ```bash
-  notation list --plain-http $IMAGE
+  notation list $IMAGE
   ```
 
   At this point, the results are empty, as there are no existing signatures
 
 ## Signing a Container Image
 
-To get things started quickly, the Notation cli supports generating self signed certificates. As you automate the signing of content, you will most likely want to create and store the private keys in a key vault. (Detailed production steps will be covered later)
+To get things started quickly, the Notation cli supports self-generated certificate chain. As you automate the signing of content, you will most likely want to create and store the private keys in a key vault. (Detailed production steps will be covered later)
```

Comment:
We can hold the changes for self-signed certificate chain, since the self-signed certificate may not be certificate chain. Let's wait until Pritesh is back next week.

Reply:
This PR is for cose branch only.